exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/dml/SelectTests.kt

```diff
@@ -284,7 +284,7 @@ class SelectTests : DatabaseTestsBase() {
     }
 
     @Test
-    fun `test encryptedColumnType with a string`() {
+    fun testEncryptedColumnTypeWithAString() {
```

Comment:
To use the contents of this test as a sample, the name format has to be changed, since KDocs can't reference a function with spaces: [KTIJ-21878](https://youtrack.jetbrains.com/issue/KTIJ-21878/KDoc-unable-to-reference-a-method-with-spaces-in-the-name).